profile-widget/divecartesianaxis.cpp

```diff
@@ -101,7 +102,7 @@ int DiveCartesianAxis::getMinLabelDistance(const DiveCartesianAxis &timeAxis) co
 	return int(ceil(interval));
 }
 
-static double sensibleInterval(double inc, int decimals, bool is_time_axis)
+static double sensibleInterval(double inc, int decimals, bool is_time_axis, bool is_metric_depth)
```

Comment:
and now we have `is_metric_depth`
not sure you should pass this as an argument as you get the answer whether it's imperial or metric from the preferences... but I'm afraid, looking at the code, that this is actually about the `three_m_based_grid`

Reply:
sensibleInterval can be called from any "Axis", and we've got quite a few.
Naming is a leftover. Initially I thought using units as an indicator to choose quantization. But then I realized that not all metric folks will be happy to see 12m instead of 10m. It has to be an option in preferences. This is why I end up having a zoo of names.

I'll fix it.